pkg/remediator/crashloopbackoff_rescheduler.go

```diff
@@ -142,7 +142,7 @@ func NewCrashLoopBackOffRescheduler(logger *zap.Logger,
 	metrics := metrics.NewCrashLoopBackOffMetrics(logger)
 	metrics.Register()
 
-	informerFactory, err := k8s.NewSharedInformerFactory(logger, filter.namespace)
+	informerFactory, err := client.GetSharedInformerFactory(logger, filter.namespace)
```

Comment:
it returns a new one each time ... so "New" not "Get"
```suggestion
	informerFactory, err := client.NewSharedInformerFactory(logger, filter.namespace)
```